### PR TITLE
DOCSP-16267 show collection type

### PR DESCRIPTION
**Pull request by davemungo**
## JIRA
https://jira.mongodb.org/browse/DOCSP-16267
[MONGOSH] Show collection type in the collections list

### STAGING
https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-16267-show-collection-type/reference/access-mdb-shell-help/#list-collections-in-the-current-database

### NOTES
There are two build errors:
ERROR(reference/access-mdb-shell-help.txt:184ish): Target not found: "label:views-landing-page"
ERROR(reference/access-mdb-shell-help.txt:199ish): Target not found: "label:views-landing-page"

This will go away when the helper code is merged into the server docs repo:
DOCSP-20658 Fix views reference link
https://github.com/10gen/docs-mongodb-internal/pull/506

### CODE REVIEW

File: source/reference/access-mdb-shell-help.txt [hunk 1]
```diff
@@ -182,6 +182,29 @@ To see the list of collections in the current database, use the
...
+   system.buckets.weather
+   system.views
+
+In the preceding example:
+
+- ``managementFeedback`` is a :ref:`view <views-landing-page>`
+- ``weather`` is a time series
+- ``survey`` is a collection
+- ``system.buckets.weather`` is a system generated collection
+  that supports the ``weather`` time series.
```
Comment (jmd-mongo) [inline source/reference/access-mdb-shell-help.txt:204]: [suggestion]

I think we can remove the full stop on line 204.

File: source/reference/access-mdb-shell-help.txt [hunk 2]
```diff
@@ -412,3 +430,11 @@ certain database commands and obtain information on your deployment:
      - Display a list of all roles, both user-defined and built-in, for
        the current database.
 
+   * - ``show tables``
+     - Print a list of collections in the current database. See ``show
```
Comment (jmd-mongo) [inline source/reference/access-mdb-shell-help.txt:434]: [suggestion]

Consider updating this to "Display" to be more consistent with usage on the rest of the page.

**Review by jmd-mongo — COMMENTED** [review]
Hi @davemungo 👋 -- thanks for the updates. I've left two suggestions for you.

Thanks!
Joe

**Review by davemungo — COMMENTED** [review]
Thanks for the feedback @jmd-mongo 

This is ready for another pass

**Review by jmd-mongo — APPROVED** [review]
Thanks for the updates. @davemungo! This one LGTM.

Thanks,
Joe

**Review by addaleax — APPROVED** [review]

**Pull request merged**